test/index.js

```diff
@@ -1,9 +1,8 @@
-var bip39 = require('../')
-var Buffer = require('safe-buffer').Buffer
+var bip39 = require('../src')
```

Comment:
nit: `../` should be fine since package.json will point to src/index.js anyways.

Reply:
https://github.com/bitcoinjs/bip39/pull/104/commits/0bc19393b07c5a99ac130dafc7f3406e59a96d01